src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -1176,6 +1176,19 @@ def test_vm_create_none_options(self, resource_group):
         ])
         self.cmd('network public-ip show -n {vm}PublicIP -g {rg}', expect_failure=True)
 
+class VMCreateMonitorTest(ScenarioTest):  # pylint: disable=too-many-instance-attributes
+
+    @ResourceGroupPreparer(name_prefix='cli_test_vm_create_with_monitor', location='centralus')
```

Comment:
too-many-instance-attributes (R0902):
Too many instance attributes (%s/%s) Used when class has too many instance attributes, try to reduce this to get a simpler (and so easier to use) class.
Do we need it?

Reply:
Good suggestion. I think we don't need this line.